pandas/core/groupby/generic.py

```diff
@@ -79,6 +79,28 @@
 if TYPE_CHECKING:
     from pandas.core.internals import Block
 
+_agg_template = """
```

Comment:
@HH-MWB can the doc decorator simplify this?

Reply:
Ok, I've used the @doc decorator, saves a line for each method. The ``**kwargs`` in @doc should not be restricted to ``str``. So I've changed it to ``Any``.

@HH-MWB , would be nice to get some examples to @doc function, took a bit of reading before I understood it.